src/Stratis.Bitcoin.Features.SmartContracts.Tests/GasInjectorTests.cs

```diff
@@ -131,7 +134,7 @@ public void TestGasInjector()
             var internalTxExecutorFactory =
                 new InternalTransactionExecutorFactory(this.keyEncodingStrategy, this.loggerFactory, this.network);
 
-            var vm = new ReflectionVirtualMachine(this.validator, internalTxExecutorFactory, this.loggerFactory, this.network, this.addressGenerator);
+            var vm = new ReflectionVirtualMachine(this.validator, internalTxExecutorFactory, this.loggerFactory, this.network, this.addressGenerator, this.assemblyLoader);
```

Comment:
Not a blocker but would be cleaner to just have this.vm, set it once in constructor, seeing as it's being used in all the methods the same way.

Reply:
Agreed